build.cake

```diff
@@ -5,8 +5,14 @@
 var target = Argument("target", "Default");
 var configuration = Argument("configuration", "Debug");
 var framework = Argument("framework", "net-4.5");
+var legacyOutputXml = Argument("LegacyOutputXml", "false");
```

Comment:
Leftover from TFS support I guess. We always want nunit3 output for our build anyway.